### PR TITLE
[2.9.0] Update csp-adapter version v104.0.0+up4.0.0

### PR DESCRIPTION
**Pull request by yarunachalam**
Description
Update version support for csp-adapter

### CODE REVIEW

**Review by martyav — APPROVED** [review]

**Comment by martyav** [comment]
Approved based purely on no typos/formatting issues

**Review by martyav — COMMENTED** [review]
I submitted an approval before seeing you had marked this as WIP. Is there more content being added besides the changes to the tables? If so, could you convert this to a draft until it's ready for review?

**Comment by jeremy-moffitt** [comment]
content looks correct to me, though we should probably wait for these two PRs to land before merging the doc PR just in case:
https://github.com/rancher/rancher/pull/46142
https://github.com/rancher/charts/pull/4206

File: versioned_docs/version-2.8/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md
Comment (btat) [inline versioned_docs/version-2.8/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md:22]:
```suggestion
```

Since this is for v2.9.0, it should be made to versioned_docs/version-2.9/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md instead. The file being modified here is intended for v2.8 listings.

**Review by btat — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.8/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md
Comment (yarunachalam) [reply versioned_docs/version-2.8/integrations-in-rancher/cloud-marketplace/aws-cloud-marketplace/install-adapter.md]: done

**Comment by martyav** [comment]
Sorry, I have to close as we have a duplicate PR open

**Comment by jeremy-moffitt** [comment]
for reference, I believe https://github.com/rancher/rancher-docs/pull/1407  is the superseding PR

**Pull request closed without merging**
